tc_aws/loaders/presigning_loader.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+
+from boto.s3.bucket import Bucket
+from thumbor.utils import logger
+from tornado.concurrent import return_future
+
+import thumbor.loaders.http_loader as http_loader
+
+from tc_aws.aws.connection import get_connection
+
+def _get_bucket(url, root_path=None):
```

Comment:
Maybe there's some factorization to do here, you could perhaps call the base loader _get_bucket method in here?